tests/unit/modules/test_gpg.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import os
+import shutil
+
+# Posix only
+try:
+    import gpg
```

Comment:
You import `gpg` here and seem to not do anything with it.

And then a bit down...

Reply:
cut and paste boilerplate that I missed when creating the test file with Shane.
I shall remove it

Reply:
Done